src/main/scala/Docker.scala

```diff
@@ -166,7 +166,9 @@ object Docker {
           )
         val http0 = new Http().configure { builder =>
           val config = builder.build()
-          val updatedProvider = config.getAsyncHttpProviderConfig match {
```

Comment:
i'm actually surprised this can be null. This modifies the default config provided by [dispatch](https://github.com/dispatch/reboot/blob/0.11.3/core/src/main/scala/execution.scala#L19-L25). It's it possible there is a bug upstream?